src/main/kotlin/no/nav/familie/ef/sak/repository/BehandlingRepository.kt

```diff
@@ -33,4 +35,17 @@ interface BehandlingRepository : RepositoryInterface<Behandling, UUID>, InsertUp
                     """)
     fun finnAktivIdent(behandlingId: UUID): String
 
+    // language=PostgreSQL
+    @Query("""
+        SELECT b.*, be.id aseksternid_id
```

Comment:
`        SELECT b.*, be.id aseksternid_id`

Kan vi fjerne alt etter komma her?

Reply:
Tja, ønsker egentlige ikke det. Hvis vi ikke har med den så blir eksternBehandlingId = 0, då risikerer vi at noen tar i bruk den og ikke får forvented ekstern_id.
Jeg tror att vi godt kan splitte disse 2 (behandling og EksternBehandlingId) fra hverendre, og så kan vi hente opp eksternBehandlingId i de få tilfellen når vi faktiskt trenger den? 👯

Reply:
aha, godt poeng. da var det bare typo du måtte fikse (som var min opprinnelige tanke)